src/tree/service.rs

```diff
@@ -120,9 +122,13 @@ pub async fn inclusion_proof<M: Middleware + 'static>(
     Ok((StatusCode::OK, Json(inclusion_proof)))
 }
 
-#[tracing::instrument(level = "debug")]
-pub async fn health() -> StatusCode {
-    StatusCode::OK
+#[tracing::instrument(level = "debug", skip(world_tree))]
+#[allow(clippy::complexity)]
+pub async fn health<M: Middleware + 'static>(
+    State(world_tree): State<Arc<WorldTree<M>>>,
+) -> Result<(StatusCode, Json<HashMap<u64, Root>>), WorldTreeError<M>> {
```

Comment:
I think the split into `(StatusCode, Json<...>)` is reduntant. The json payload alone implies code 200

Reply:
Sounds good, I can remove the `StatusCode`,